.github/workflows/deploy.yml

```diff
@@ -20,49 +29,37 @@ jobs:
         run: |
           git config --global user.name "github-actions[bot]"
           git config --global user.email "github-actions[bot]@users.noreply.github.com"
-      - name: Get tag name
-        id: tag
-        run: |
-          tag=$(basename "${{ github.ref }}")
-          echo "tag=$tag" >> $GITHUB_OUTPUT
-      - name: Set up Node
-        uses: actions/setup-node@v3
-        with:
-          node-version: '18'
       - name: Install Emscripten
         uses: mymindstorm/setup-emsdk@v14
         with:
-          version: ${{ env.EMSCRIPTEN_VERSION }}
-      - name: Install dependencies
-        run: npm install
+          version: ${{env.EMSCRIPTEN_VERSION}}
+      - name: Set up tree-sitter
+        uses: tree-sitter/setup-action/cli@v1
       - name: Compile grammar
-        run: npm run generate
+        run: tree-sitter generate --no-bindings
       - name: Build Wasm binary
-        run: npm run build-wasm
+        run: tree-sitter build --wasm
```

Comment:
Hm, I worry about the `tree-sitter` version here drifting apart from the version declared in the `package.json`. Does this always use the newest version or does it check the `package.json`?

Also, I feel that we are now duplicating the build commands in a few different places, I'd prefer to keep them centralized in the `package.json`, we'll need node and npm anyways from what I can tell.

Reply:
It uses the latest version but it's also possible to use the one in `package.json`.